src/components/Communication/Layout/Layout.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Flex, FlexItem, Button, Text } from "@fluentui/react-northstar";
+import { Illustration } from "../Illustration";
+
+export interface ILayout {
+  title?: string;
+  desc?: string;
+  image?: React.ReactNode;
+  actions?: ILayoutActions;
+}
+
+export interface ILayoutActions {
+  primary?: ILayoutAction;
+  secondary?: ILayoutAction;
+  tertiary?: ILayoutAction;
+}
+
+export interface ILayoutAction {
+  label: string;
+  action: () => void;
+}
+
+export const Layout = ({ title, desc, image, actions }: ILayout) => (
+  <Flex
+    hAlign="center"
+    gap="gap.large"
+    column
+    style={{
+      width: "100%",
+      maxWidth: "33.5rem",
+      margin: "3rem 1.25rem",
+    }}
+  >
+    {image && <Illustration image={image} />}
```

Comment:
Shouldn't we allow consumers to pass another type of image besides svg, like png or jpg?

Reply:
```
export const Illustration = ({ image }: { image: React.ReactNode }) => (
  <Flex className="illustration-container" vAlign="center" hAlign="center">
    {image}
  </Flex>
);
```

yes, they could do that too

Reply:
The only issue we might run into here is that this expects a `ReactNode`, not just a string, so this could not be configured using plain JSON as with the other components.

Maybe illustrations could be named, like we do for icons in `lib/Icon.tsx`.